### PR TITLE
home style

### PR DESCRIPTION
**Pull request by aekix**
home style

### CODE REVIEW

File: frontend/src/components/pages/Home/Home.js [hunk 1]
```diff
@@ -5,13 +5,24 @@ class Home extends React.Component {
     render = () => {
         return (
             <>
-                <ul>
-                    <li><Link to="/create-room">Create room</Link></li>
-                    <li><Link to="/join-room">Join room</Link></li>
-                </ul>
+                <div className="label-title">LE QUIZ
```
Comment (alexisljn) [inline frontend/src/components/pages/Home/Home.js:8]: Why a div if it's just for the main title ? Plus maybe we gonna use the logo we have to talk about that and we have special component for this purpose (watch my comment in style.css)

File: frontend/src/css/style.css [hunk 2]
```diff
@@ -671,3 +671,65 @@ button:disabled {
...
+    display: flex;
+    justify-content: center;
+    margin-top: 200px;
+}
+
+.home-menu-wrapper {
+    width: 50%;
+}
+
+.btn-create-room {
```
Comment (alexisljn) [inline frontend/src/css/style.css:685]: If it's not in util.css, our classes shouldn't use bootstrap convention name

File: frontend/src/css/style.css [hunk 3]
```diff
@@ -671,3 +671,65 @@ button:disabled {
...
+    cursor: pointer;
+
+}
+
+.home-menu-label {
+    font-size:3em;
+    color: white;
+}
+
+.label-title {
```
Comment (alexisljn) [inline frontend/src/css/style.css:726]: We have a special component called title for the displaying of page's main title

File: frontend/src/components/pages/Home/Home.js [hunk 4]
```diff
@@ -5,13 +5,24 @@ class Home extends React.Component {
...
+                </div>
+                <div className="home-menu">
+                    <div className="home-menu-wrapper">
+                        <button className="btn-create-room btn-menu" onClick={() => document.location.href = "/create-room"}>
```
Comment (alexisljn) [inline frontend/src/components/pages/Home/Home.js:12]: It's not the way we redirect users in the app, you should use the history object

File: frontend/src/components/pages/Home/Home.js [hunk 5]
```diff
@@ -5,13 +5,24 @@ class Home extends React.Component {
...
+                            <p className="home-menu-label">Créer un Salon</p>
+                        </button>
+                    </div>
+                    <div className="home-menu-wrapper">
+                        <button className="btn-join-room btn-menu" onClick={() => document.location.href = "/join-room"}>
```
Comment (alexisljn) [inline frontend/src/components/pages/Home/Home.js:17]: It's not the way we redirect users in the app, you should use the history object

**Review by alexisljn — CHANGES_REQUESTED** [review]
Design is not responsive, use flexbox logic pls. Our design perfectly fit with this philosophy

**Review by alexisljn — APPROVED** [review]

**Comment by aekix** [comment]
With these changes the responsive is broken. Can you fix it?

**Review by alexisljn — CHANGES_REQUESTED** [review]
Responsive broken. I'm on it

**Comment by alexisljn** [comment]
It's done

**Review by alexisljn — APPROVED** [review]

**Pull request merged**